src/Dropdown.php

```diff
@@ -4,359 +4,657 @@
-     * - url: string|array, optional, the URL of the item link. This will be processed by {@see currentPath}.
-     *   If not set, the item will be treated as a menu header when the item has no sub-menu.
-     * - visible: bool, optional, whether this menu item is visible. Defaults to true.
-     * - linkOptions: array, optional, the HTML attributes of the item link.
-     * - options: array, optional, the HTML attributes of the item.
-     * - items: array, optional, the submenu items. The structure is the same as this property.
-     *   Note that Bootstrap doesn't support dropdown submenu. You have to add your own CSS styles to support it.
-     * - submenuOptions: array, optional, the HTML attributes for sub-menu container tag. If specified it will be
-     *   merged with {@see submenuOptions}.
-     *
-     * To insert divider use `-`.
+     * Sets the CSS classes for the dropdown container.
+     *
+     * @param BackedEnum|string ...$values The CSS class for the dropdown container.
+     *
+     * @return self A new instance with the specified CSS class for the dropdown container.
      */
-    public function items(array|string|Stringable|null $value): self
+    public function containerClasses(BackedEnum|string ...$values): self
+    {
+        $new = clone $this;
+        $new->containerClasses = $values;
+
+        return $new;
+    }
+
+    /**
+     * Set the direction of the dropdown component.
+     *
+     * @param DropdownDirection $value The direction of the dropdown component.
+     *
+     * @return self A new instance with the specified direction of the dropdown component.
+     */
+    public function direction(DropdownDirection $value): self
+    {
+        $new = clone $this;
+        $new->containerClasses = [$value];
+
+        return $new;
+    }
+
+    /**
+     * List of links to appear in the dropdown. If this property is empty, the widget will not render anything.
+     *
+     * @param array $value The links to appear in the dropdown.
+     *
+     * @return self A new instance with the specified dropdown to appear in the dropdown.
+     *
+     * @psalm-param DropdownItem[] $value The links to appear in the dropdown.
+     */
+    public function items(DropdownItem ...$value): self
     {
         $new = clone $this;
         $new->items = $value;
-        $new->tagName = null;
 
         return $new;
     }
 
     /**
-     * Set dropdown alignment
-     * @see https://getbootstrap.com/docs/5.3/components/dropdowns/#menu-alignment
+     * Sets the theme for the dropdown component.
+     *
+     * @param string $value The theme for the dropdown component.
+     *
+     * @return self A new instance with the specified theme.
+     */
+    public function theme(string $value): self
+    {
+        return $this
+            ->addAttributes(['data-bs-theme' => $value === '' ? null : $value])
+            ->toggleId($this->toggleId === false ? true : $this->toggleId);
```

Comment:
Why need to change toggle ID on set theme?

Reply:
Why change `toggleId` on change theme? It's very strange.

Reply:
https://getbootstrap.com/docs/5.3/customize/color-modes/#dark-mode

Reply:
Can you explain why need to set toggle ID when we change theme only?

Reply:
By default the dropdown has no id in the toggle it is not necessary, but when configuring the theme it is necessary to add the attribute `aria-labelledby` check the link above.

Reply:
Without toggle ID doesn't theme work?

Reply:
The idea is to respect the bootstrap5 documentation, and in this case, the `aria-labelledby` property is applied.

Reply:
It's example only. Seems, docs don't contain requirements to add `aria-labelledby` and toggle ID.

Reply:
Of course, i can remove the behavior, but the example clearly uses it.

Reply:
done.